Formula/ne.rb

```diff
@@ -18,9 +18,21 @@ def install
 
   test do
     ENV["TERM"] = "xterm"
```

Comment:
It's "related", so I'd rather just do it here than go through the trouble of opening and pulling another one...